test/integration/scheduler/taint_test.go

```diff
@@ -689,7 +682,6 @@ func TestTaintBasedEvictions(t *testing.T) {
 				50,               // Large cluster threshold
 				0.55,             // Unhealthy zone threshold
 				true,             // Run taint manager
```

Comment:
Need to update

```
staging/src/k8s.io/api/core/v1/well_known_taints.go:    // and feature-gate for TaintBasedEvictions flag is enabled,
staging/src/k8s.io/api/core/v1/well_known_taints.go:    // and feature-gate for TaintBasedEvictions flag is enabled,
```

Reply:
The TaintBasedEvictions feature-gate is node removed now (it'll be removed in v1.19). It's move to ga and locked to true.
So may be it's better to update this comments when we totally remove this feature-gate.
What do you think?

Reply:
Got it, thx @skilxn-go 's reply!

Reply:
After think about your comment again, I think it's ok to do the clean up work now. So I fixed them, thanks for your review 😄

Reply:
Can you also update the comments related with `TaintNodesByCondition`? (that feature has been graduated)

Reply:
Yes, and it's done now